Makefile

```diff
@@ -47,7 +47,7 @@ clean: ##@prepare Remove all output folders
 
 clean-nix: SHELL := /bin/sh
 clean-nix: ##@prepare Remove complete nix setup
-	sudo rm -rf /nix ~/.cache/nix
```

Comment:
If we are purging nix completely, should't we also remove `~/.nix-profile`, `~/.nix-defexpr`, and `~/.nix-channels`
?

Reply:
yes!